tmt/base.py

```diff
@@ -3119,8 +3116,6 @@ def resolve_dynamic_ref(
         if not os.path.exists(ref_filepath):
             raise tmt.utils.FileError(
                 f"Dynamic 'ref' definition file '{ref_filepath}' does not exist.")
-    else:
-        ref_filepath = os.path.join(workdir, DEFAULT_DYNAMIC_REF_FILEPATH)
```

Comment:
This will probably result in an error, the `ref_filepath` variable may now be undefined for the upcoming if.

Reply:
Right you are.. Those conditions can be simplified even further anyway.